app/src/main/java/at/aau/edu/wizards/gameModel/GameModelPlayer.kt

```diff
@@ -0,0 +1,49 @@
+package at.aau.edu.wizards.gameModel
+
+import kotlin.math.abs
+
+class GameModelPlayer(
+    val id: Int,
+    private val dealer: GameModelDealer,
+    val isHuman: Boolean
+) {
+
+    val cards = ArrayList<GameModelCard>()
+    val scores = ArrayList<Int>()
+    val guesses = ArrayList<Int>()
+
+    fun dealCards(amount: Int) {
+        for (card in 1..amount) {
+            cards.add(dealer.dealCardInSet())
+        }
+    }
+
+    fun hasColor(color: GameModelCard.Color): Boolean {
+        for (card in cards) {
+            when (card) {
```

Comment:
Personally, I don't think loc (lines of code) is too useful of a grading tool. For example this code was completely revised 3 times and that doesn't show up as jack shit in loc. I think the point here is just to make sure everyone actually does a considerable part of the finished project.

I think the one-liner is considerably more readable. It is written this way because I didn't know you could write it that way. I will go trough the code again and try to switch out similar cases.

Reply:
Including above line, found 2 similar cases and changed them.